proto/evmos/epochs/genesis.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package evmos.epochs.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/tharsis/evmos/x/epochs/types";
+
+message EpochInfo {
+  string identifier = 1;
```

Comment:
@ramacarlucho what is the identifier for? Can we omit this or do we need it for something?

Reply:
Yes, you need to identify the epoch in case theres more than one. 
Since other modules can add different epochs, you need to check if the epoch triggered is the one you were expecting.